### DISCUSSION TITLE
Should the main gutenberg plugin page still be the demo?

### DISCUSSION THREAD

**Discussion by aaronjorbin** [WordPress/gutenberg#58327] [General]
While having the demo absolutely made sense during the early stages of Gutenberg, the plugin has evolved to be a place for (using terms from the readme) "tech-savvy early adopter" to test "bleeding-edge and experimental features".

Based on this, I think that the experiments page should be the default rather than the example page.

**Comment by t-hamano** [comment]
When you say "Gutenberg plugin main page", do you mean this Demo menu? I'm not sure what you mean by "main page", "default page" 🤔

![demo-menu](https://github.com/WordPress/gutenberg/assets/54422211/3d2c3bfb-c4ed-4c8d-9f1e-7a7325e111f0)

**Comment by aaronjorbin** [comment]
I mean the menu, not the sub-menu.  The one added through https://github.com/WordPress/gutenberg/blob/trunk/lib/init.php#L16-L23

**Comment by t-hamano** [comment]
Understood. This menu was probably first added in #1065. At the time, the default editor was still the classic editor, which might have been added to demonstrate the block editor.

I think it's okay to use the Experiments page as the main page, and even delete the demo page.
